duckduckgo/macos-browser
# Use updated BSK branch

Task/Issue URL: https://app.asana.com/0/1202198075905303/1202519457279975/f
Tech Design URL:
CC:

**Description:**
This PR updates AMP deep extraction CBRs to block script load types.

**Steps to test this PR:**
1. Visit https://privacy-test-pages.glitch.me/privacy-protections/amp/
2. Ensure the links on this page take you to the expected sites.

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
